roottest/root/io/json/runSTL0.C

```diff
@@ -1,41 +1,29 @@
-{
-// Fill out the code of the actual test
-#ifndef SECOND_RUN
-   gROOT->ProcessLine(".L test_classes.h+");
-#endif
+#include "test_classes.h"
 
-#if defined(ClingWorkAroundMissingDynamicScope) && !defined(SECOND_RUN)
-#define SECOND_RUN
-   gROOT->ProcessLine(".x runSTL0.C");
-#else
+void runSTL0()
+{
+   gSystem->Load("libJsonTestClasses");
 
    TJsonEx7 ex7;
```

Comment:
This is odd.  The compilation by Cling of `runSTL0` happens before the `gSystem->Load` is executed, so it looks like the test is relying on auto-loading of the library instead.  Is that the intent?  Does it still work when ROOT is compiled with `runtime_cxxmodule=OFF`?

Reply:
I guess, we will see it in CI

Reply:
> Library only needed to get generated streamer infos - 

Fair enough, I now understand what you mean.  I see that all the the `TJSon...` classes are 'header' only classes and the library only contains the dictionary (source) files, thus the function's code can be 'compiled' and linked without the library.